core/resp.go

```diff
@@ -228,6 +228,17 @@ func Encode(value interface{}, isSimple bool) []byte {
 			buf.Write(Encode(row.Value.Value, false))
 		}
 		return []byte(fmt.Sprintf("*%d\r\n%s", len(v), buf.Bytes()))
+	case HashMap:
```

Comment:
Is this being used somewhere yet?

Reply:
Yup! To encode the map and then create the object that is finally stored.
https://github.com/kolharsam/dice/blob/0e83d3f102747013d63d5e416150d82431ad521b/core/eval.go#L2353